tests/data/fmi2/quarter_truck/README.md

```diff
@@ -0,0 +1,10 @@
+# FMI 2.0 test FMUs
+
+| Name                   | Origin         | License                       |
+| ---------------------- | -------------- | ----------------------------- |
+| `Chassis.fmu`  | OSP            | [MIT](LICENSE)   |
+| `Wheel.fmu` | OSP            | [MIT](LICENSE)   |
+| `Ground.fmu`           | [MIT] | [MIT](LICENSE) |
```

Comment:
The spacing here is weird. Also, cell 3:2 should probably not contain `[MIT]`.

Reply:
Cleaned this up & removed ground. LICENSE was taken from the FMICPP repo, which appears to be MIT.